src/main.js

```diff
@@ -208,17 +246,43 @@ const outputHTML = envSpecEntriesArray => {
     //return as string value
     return envSpecEntriesToPrint.join("");
   }
-  //in case of syntax error , print HTML format
-  return "Error:Wrong Syntax";
+  //in case of syntax error ,
+  //return "Error:Wrong Syntax";
 };
 
 /** @function envSpecToHTML
- * @desc gives final output using {@link outputHTML} and {@link checkValidationOfValues}
+ * @desc gives final output using {@link parse} , {@link outputHTML} and {@link checkValidationOfValues}
  * @param {string} envSpec the .env file given in string format
- * @returns {string} HTML code
+ * @returns {promise} that resolves to string
  */
 const envSpecToHTML = envSpec => {
-  return outputHTML(checkValidationOfValues(envSpec));
+  return parse(envSpec)
```

Comment:
We do not need `envSpecToHTML` anymore, now that we have the promise based `parse`.